state.go

```diff
@@ -1075,11 +1075,13 @@ func (e *RebootState) Handle(ctx *StateContext, c Controller) (State, bool) {
 
 type RollbackState struct {
 	UpdateState
+	swap bool
 }
 
-func NewRollbackState(update client.UpdateResponse) State {
+func NewRollbackState(update client.UpdateResponse, swapPartitions bool) State {
```

Comment:
This was added in subsequent commit which maybe should be squashed with this one. Anyway, we are using this in more places with the argument set to `false`.